src/google/cloud/ndb/query.py

```diff
@@ -913,7 +951,7 @@ class Query:
         filters (Union[Node, tuple]): Node representing a filter expression
             tree. Property filters applied by this query. The sequence
             is ``(property_name, operator, value)``.
-        orders (Union[QueryOrder, list]): The field names used to
+        orders (Union[tuple, list]): The field names used to
```

Comment:
What do you think of making this `order`, and just leaving in `orders` as a synonym for backwards compatibility? The unnecessary plural is kind of ugly.

Reply:
Completely agree. Will make the change.